src/util/router/makePaginatedRoute.js

```diff
@@ -2,30 +2,32 @@ import makeRoute from './makeRoute'
 
 /**
  * @param {object} params
+ * @param {string} params.route
  * @param {string|number|null} params.author
  * @param {string|number|null} params.category
  * @param {string|number|null} params.page
  * @returns {string}
  */
-export default function makeBlogRoute (params) {
+export default function makePaginatedRoute (params) {
```

Comment:
If we refactor to `onMake{Next,Prev}Route`, we probably can revert this to the original `makeBlogRoute`.

Because short links to these routes exist and must be maintained, this route structure for blog posts are set in stone.

New routes like what we're doing here don't need to follow this strictly, however, so we could just use a query param for pagination on leaderboard